clang/test/Sema/nullptr.c

```diff
@@ -108,3 +108,6 @@ void test_f1() {
   int ir = (f1)(nullptr);
 }
 
+// __nullptr keyword in C
```

Comment:
Maybe also add these:
```c
static_assert(nullptr == __nullptr);
static_assert(_Generic(typeof(__nullptr), nullptr_t: true, default: false));
```
Just so we have a few more checks. I don’t think we need to go as far as to copy-paste the rest of this file for this.

(I don’t write these `_Generic` selections too often so I hope I got that one right.)